pkgs/servers/home-assistant/parse-requirements.py

```diff
@@ -156,8 +165,11 @@ def main() -> None:
             # Some requirements are specified by url, e.g. https://example.org/foobar#xyz==1.0.0
             # Therefore, if there's a "#" in the line, only take the part after it
             req = req[req.find("#") + 1 :]
-            name = req.split("==")[0]
+            name, hass_version = req.split("==", maxsplit=1)
```

Comment:
```suggestion
            name, required_version = req.split("==", maxsplit=1)
```

Reply:
Ah yeah, that is indeed a bit ambiguous.